R/get_coverage_wcs.R

```diff
@@ -46,35 +47,56 @@
 #' }
 #'
 get_coverage_wcs <- function(
-    wcs = c("dtm", "dsm", "omz", "omw"),
+    wcs = c("dtm", "dsm", "omz", "omw", "dhmv"),
     bbox,
     layername,
     resolution,
-    wcs_crs = "EPSG:4258",
+    wcs_crs = c("EPSG:4258", "EPSG:31370"),
```

Comment:
I think it is crucial that the correct EPSG code is passed here. This is the CRS that is used in the WCS to store the raster data. I noticed that using the default EPSG:4258 with DHMV resulted in raster image that was shifted compared to the corresponding wcs = "dtm" alternative.

Reply:
Interesting. Yet shift alone does not indicate which one is wrong.
And actually, I don't understand why we should make the "input" crs explicit. Shouldn't the web API know by itself what crs is used, return it, and only upon user request apply transformation to an `output_crs`?